cognite/transformations_cli/commands/deploy/transformation_types.py

```diff
@@ -69,7 +75,7 @@ class TransformationConfig:
     name: str
     query: Union[str, QueryConfig]
     authentication: Union[AuthConfig, ReadWriteAuthentication]
-    schedule: Optional[str]
+    schedule: Optional[Union[str, ScheduleConfig]]
```

Comment:
Can we change ScheduleConfig to avoid the Union here?

Reply:
It will break backward compatibility in case anyone started using this.

Reply:
if user doesn't need pause option it will be `schedule: * * * * *`